Add Grasping Deep Learning post (#512)
ros-planning/moveit.ros.org: _posts/2020-09-28-grasp-deep-learning.md

description: Grasping using Deep Learning now available in the MoveIt Task Constructor
categories:
- Deep Learning
- Grasping
- MoveIt
- 3D perception
---

[//]: # (Image References)
[image2]: /assets/images/blog_posts/deep_learning_grasp/image2.gif
[image3]: /assets/images/blog_posts/deep_learning_grasp/image3.gif

MoveIt now supports robust grasp pose generation using deep learning. Pick and place robots equipped with a depth camera and either a parallel jaw or suction gripper can increase productivity when paired with deep learning. The MoveIt Task Constructor provides an interface for any grasp pose generation algorithm making MoveIt’s pick and place capabilities more flexible and powerful.

Currently, the [Grasp Pose Detection](https://github.com/atenpas/gpd) (GPD) library and [Dex-Net](https://berkeleyautomation.github.io/dex-net/) are being used to detect 6-DOF grasp poses given 3D sensor data. GPD is capable of generating grasp poses for parallel jaw grippers and Dex-Net works with both parallel jaw and suction grippers. These neural networks are trained on datasets containing millions of images allowing them to pick novel objects from cluttered scenes.

> The depth camera can either mount to a link on the robot or remain stationary. If the camera is mounted to a link or if multiple cameras are used, it is possible to reconstruct a 3D point cloud or collect depth images from multiple viewpoints. This technique enables grasp pose generators to sample more grasp candidates from views that would otherwise be occluded.

Reviewer: ```suggestion
The depth camera can either be mounted to a robot link, or be stationary in the environment. If the camera is mounted on the robot or multiple cameras are used, it is possible use multiple viewpoints to reconstruct a 3D point cloud or collect depth images. This technique allows sampling grasp candidates from views that would otherwise be occluded.
```
Just tightening this up because the first and second sentence used mount/mounted in different ways.
